utils.py

```diff
@@ -62,11 +62,12 @@ def get_requested_llvm_version():
 
 def get_build():
     arch = os.environ["ARCH"]
-    config = os.environ["CONFIG"]
+    configs = sorted(os.environ["CONFIG"].split(","))
     llvm_version = get_requested_llvm_version()
     for build in _read_builds():
-        if build["target_arch"] == arch and build["kconfig"][0] == config \
-                and build["toolchain"] == llvm_version:
+        if build["target_arch"] == arch and \
+           build["toolchain"] == llvm_version and \
+           sorted(build["kconfig"]) == configs:
```

Comment:
how come we sort these configs?

Reply:
Fair point, the lists are created in the same order so sorting shouldn't be necessary. I will test without that change later today.